back/engines/commercial/user_custom_fields/spec/acceptance/ref_distributions_spec.rb

```diff
@@ -0,0 +1,159 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'rspec_api_documentation/dsl'
+
+resource 'Representativeness reference distributions' do
+  header 'Content-Type', 'application/json'
+
+  shared_examples 'unauthorized requests' do
+    context 'when visitor' do
+      example 'returns 401 (Unauthorized)', document: false do
+        do_request
+        expect(status).to eq(401)
+      end
+    end
+
+    context 'when normal user' do
+      before { user_header_token }
+
+      example 'returns 401 (Unauthorized)', document: false do
+        do_request
+        expect(status).to eq(401)
+      end
+    end
+  end
+
+  get 'web_api/v1/users/custom_fields/:custom_field_id/reference_distribution' do
+    let!(:reference_distribution) { create(:ref_distribution) }
+    let(:custom_field_id) { reference_distribution.custom_field.id }
+
+    context 'when admin' do
+      before { admin_header_token }
+
+      context 'when the custom field has a reference distribution' do
+        let(:expected_response) do
+          {
+            data: {
+              id: reference_distribution.id,
+              type: 'reference_distribution',
+              attributes: {
+                distribution: reference_distribution.probabilities_and_counts.symbolize_keys
+              },
+              relationships: {
+                values: {
+                  data: reference_distribution.distribution.keys.map do |key|
+                    { type: 'custom_field_option', id: key }
+                  end
+                }
+              }
+            }
+          }
+        end
+
+        example_request 'Get the reference distribution associated to a custom field' do
+          expect(status).to eq(200)
+          expect(json_response_body).to match(expected_response)
+        end
+      end
+
+      context 'when the custom field does not have a reference distribution' do
+        before { reference_distribution.destroy! }
+
+        example_request 'returns 404 (Not Found)' do
+          expect(status).to eq(404)
+        end
+      end
+    end
+
+    include_examples 'unauthorized requests'
+  end
+
+  post 'web_api/v1/users/custom_fields/:custom_field_id/reference_distribution' do
+    parameter :distribution, 'The distribution as (unnormalized) counts', required: true
```

Comment:
```suggestion
    parameter :distribution, 'The distribution as (non normalized) counts', required: true
```

Reply:
`unnormalized` is the most common way of saying it in statistics, maybe bc "non normalized" looks a lot like "non normal" which is really different (but that's just a guess). I'll rework the description of the parameter though, as latest changes in the definition of the score (or lack of) add some more constraints to the counts.

Reply:
> is the most common way of saying it in statistics

I did not know. "unnormalized" does not seem to be an English word. That is why I made the suggestion.